packages/inventory/src/components/InventoryList/tests/index.test.tsx

```diff
@@ -84,7 +84,7 @@ describe('Inventory list Page', () => {
     expect(fetch.mock.calls[0]).toEqual(fetchCalls[3]);
 
     expect(wrapper.text()).toMatchInlineSnapshot(
-      `"Inventory items+ Add new inventory itemProduct nameQtyPO no.Serial no.Delivery dt.Acct. end dt.Unicef sectionDonorActionsChange name 11101123434Jan 02, 2020May 02, 2021HealthADBEditChange name 21101123434Feb 02, 2020May 02, 2021HealthADBEdit"`
+      `"Inventory items+ Add new inventory itemProduct nameQtyPO no.Serial no.Delivery dt.Acct. end dt.Unicef sectionDonorActionsNo Data"`
```

Comment:
@HaseemUlHaq could we add another test suite(it block) like this one, where we actually get some inventories for the mocked service point,

Reply:
Seems like the test case does not much the description `it('renders when data is present'`